### PR TITLE
Fetch Path from HereApi

### LINKED ISSUE #2629: Implement requests for google speeds api
Api description
https://developers.google.com/maps/documentation/roads/speed-limits

### PR DESCRIPTION
**Pull request by carloscaldas**
Closes #2629 

This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/2633)

### CODE REVIEW

**Comment by carloscaldas** [comment]
test!

**Comment by carloscaldas** [comment]
test!

File: build.gradle [hunk 1]
```diff
@@ -223,6 +223,8 @@ dependencies {
     compile group: 'io.circe', name: "circe-parser_${scalaBinaryVersion}", version: circeBinaryVersion
 
     compile group: 'com.typesafe.play', name: "play-json_${scalaBinaryVersion}", version: '2.6.3'
+    compile group: 'com.typesafe.play', name: 'play-ahc-ws-standalone_2.12', version: '2.0.3'
```
Comment (JustinPihony) [inline build.gradle:226]: Don't we already have an external http library?

Comment (JustinPihony) [reply build.gradle:226]: Yah, we actually have 2 - Akka HTTP and Apache Httpcomponents Fluent-HC. We probably shouldn't add a third...and in fact should reduce to one.

Comment (carloscaldas) [reply build.gradle:226]: this task was supposed to be implemented urgently. so... there was no time to think about this

**Pull request merged**